tests/hypothesis_strategies/parameters.py

```diff
@@ -2,6 +2,7 @@
 
 import hypothesis.strategies as st
 import numpy as np
+from altair import Optional
```

Comment:
dont know this can you explain? any chance the autoimport went wrong and should use typing?

Reply:
ahem .... just me getting accustomed to using VScode 🤣  thanks for the catch, fixed it